docs/user-guide/deploy_etcd.md

```diff
@@ -7,10 +7,12 @@ The following prerequisites need to be met.
 * A [Senza Definition](http://stups.readthedocs.org/en/latest/components/senza.html#senza-definition)
 * A template can be found here: [etcd-appliance.yaml](https://github.com/zalando/spilo/blob/master/etcd-cluster-appliance/etcd-cluster.yaml)
 * The Security Group defined in the Senza Definition needs to be created
+* The appliction\_id `etcd-cluster` needs to be registered in yourturn
+* The S3 mint bucket needs to be added to the Access Control
 
 To deploy the etcd-appliance, use the following:
 
-	senza create DEFINITION.yaml VERSION HOSTED_ZONE DOCKER_IMAGE
+	senza create DEFINITION.yaml VERSION HOSTED_ZONE DOCKER_IMAGE MINT_BUCKET
```

Comment:
Should we specify how to create mint_bucket in senza? Or provide a custom template to generate it, as well as other parameters for etcd (I think in senza one can use custom templates w/o adding them to senza itself).